### ISSUE TITLE
Titles appear as installed after system transfer

### ISSUE THREAD

**Issue by ShaneDrury** [Cruel/freeShop#176]
After a system transfer (o3DS to n3DS) titles installed with freeShop do not appear on the home screen but do appear as installed titles in freeShop. 
I presume this is because the title files are still on the sd card but haven't actually been installed/registered with the n3DS.
The method of system transfer was the "PC" one where you manually transfer files over to a new sd card.

A possible solution is to delete and reinstall a title but this also deletes the save game. The save games are still on the sd card so it would be nice if there was an option to reinstall a title. I'm unsure of the mechanism of installation but if this doesn't touch the save game files it might solve this particular edge case.

**Comment by Cruel** [comment]
I'm not too familiar with system transfers. Try looking at title list in FBI. I'm assuming the titles are also listed there? And can they not be launched?

**Comment by ShaneDrury** [comment]
The titles seem to appear in the FBI menu without their proper names/icons. It just gives the codes e.g. "CTR-P-AMTE" instead. I cannot launch them from this menu "The SD Card was removed".

**Comment by Cruel** [comment]
I believe you can import installed titles, but it's difficult for me to test myself to implement it. I can guess in the meantime you can use JK Save Manage to get your saves and reinstall the titles you want.

**Comment by ShaneDrury** [comment]
Yeah JKSM works for the minute. Export save, uninstall, reinstall title, import save. A bit arduous but it works for now!

**Comment by quantumcat1** [comment]
I was going to make an issue to do with this too. It would be great if you could have the option to reinstall tickets even if marked as installed (maybe as a switchable option since most of the time you don't want your ticket reinstalled if you accidentally try to install a game you already have).
 I was trying to get one game to appear again after a transfer, and thought I could just get freeShop to install it, but that didn't work. It also didn't work when I tried installing from QR code from the title database site for some reason, so I ended up using tikShop but that reenabled like 30 games.

**Comment by deadair3210** [comment]
Try using something like FBI to reinstall the ticket for the affected item. Maybe the ticket got messed up somehow? Had a issue with games not appearing on the HOME Menu when I restored a old NAND backup, reinstalling the tickets fixed it

**Comment by Cruel** [comment]
Oh, if it just needs a reinstall of ticket, that could be something easy to add. I mean, I could automatically install missing tickets on launch, but is there any case when people wouldn't want that to happen?

**Comment by deadair3210** [comment]
No reason that I can think of, since HOME Menu will actively refuse to show any app that has either a missing or corrupted ticket. Possibly have a button that could be used to trigger a sort of repair operation where the app reinstalls any tickets that it knows about that arent system titles.

**Comment by deadair3210** [comment]
I don't know if any issues with reinstalling the ticket to an app either, when I reinstalled my tickets, all saves were there and the icons were even in their customized spot on the home menu if it had one

**Comment by Cruel** [comment]
Ok, I'll rename this issue and add it later.
